### PR TITLE
 auth forms.py code reformat

### PR DESCRIPTION
**Pull request by annagav**

### CODE REVIEW

File: auth/forms.py [hunk 1]
```diff
@@ -2,51 +2,37 @@
...
-		import os
-		import pudb
-# 		pudb.set_trace()
-		if(self.data.get('account_type') == 'student'):
-			create_course_records(request,self.data.get('CC'))
-# 		elif(self.data.get('account_type') == 'instructor'):
-		add_to_group(request, self.data.get('account_type'))
+
+    def save(self, request):
+        print request
```
Comment (carsongee) [inline auth/forms.py]: please remote the prints.  If you need to log, please use logging, i.e.

``` python
...
import logging
...

log = logging.getLogger(__name__)
...
...
log.debug('SignupForm received request: %r', request)
...
```

File: auth/forms.py [hunk 2]
```diff
@@ -2,51 +2,37 @@
...
 
+def create_course_records(usr, cc_input):
+    cc_input = cc_input.replace(' ', '')
+    course_codes = cc_input.split(';')
+    for course_code in course_codes:
+        if Course.objects.filter(code=course_code).count() > 0:
+            course = Course.objects.get(code=course_code)
+            user_course = UserCourse(course_name=course_code, user=usr, courseID=course)
+            user_course.save()
+            print course_code
```
Comment (carsongee) [inline auth/forms.py]: same here

**Comment by annagav** [comment]
removed print statements.

File: auth/forms.py [hunk 3]
```diff
@@ -2,51 +2,37 @@
...
+
+    def save(self, request):
+        print request
+        if self.data.get('account_type') == 'student':
+            create_course_records(request, self.data.get('CC'))
+        add_to_group(request, self.data.get('account_type'))
         """
         An extension point for subclasses.
         """
         pass
```
Comment (carsongee) [inline auth/forms.py]: can you remove this pass and docstring.  They are both pointless

File: auth/forms.py [hunk 4]
```diff
@@ -2,51 +2,35 @@
...
+def create_course_records(usr, cc_input):
+    cc_input = cc_input.replace(' ', '')
+    course_codes = cc_input.split(';')
+    for course_code in course_codes:
+        if Course.objects.filter(code=course_code).count() > 0:
+            course = Course.objects.get(code=course_code)
+            user_course = UserCourse(course_name=course_code, user=usr, courseID=course)
+            user_course.save()
 
+        # get the data
```
Comment (carsongee) [inline auth/forms.py]: and remove this...

**Comment by carsongee** [comment]
sorry for the long delay between viewings, I had to let this sink in a bit.

**Comment by annagav** [comment]
Done!

**Comment by carsongee** [comment]
:fire:

**Pull request merged**